packages/framework/resources/views/layouts/navigation.blade.php

```diff
@@ -1,9 +1,9 @@
 @php
-    $navigation = \Hyde\Framework\Features\Navigation\NavigationMenu::create();
+    $navigation = app('navigation.main');
 @endphp
 
 <nav aria-label="Main navigation" id="main-navigation" class="flex flex-wrap items-center justify-between p-4 shadow-lg sm:shadow-xl md:shadow-none dark:bg-gray-800">
-    <div class="flex flex-grow items-center flex-shrink-0 text-gray-700 dark:text-gray-200">
+    <div id="main-navigation-brand" class="flex flex-grow items-center flex-shrink-0 text-gray-700 dark:text-gray-200">
```

Comment:
While we're breaking stuff, maybe we should normalize these ID tags and their locations.